packages/ts-migrate-plugins/src/plugins/explicit-any.ts

```diff
@@ -7,15 +7,20 @@ import { AnyAliasOptions, validateAnyAliasOptions } from '../utils/validateOptio
 
 type Options = AnyAliasOptions;
 
+export interface LintConfig {
+  useTabs: boolean;
+  tabWidth: number;
+}
+
 const explicitAnyPlugin: Plugin<Options> = {
   name: 'explicit-any',
 
-  run({ options, fileName, text, getLanguageService }) {
+  run({ options, fileName, text, getLanguageService }, lintConfig?: LintConfig) {
```

Comment:
could we move `lintConfig` into the plugin options?

Reply:
seemed more painful.. also this pr is not quite ready.. i guess i accidentally submitted it,, lintConfig also needs to be added to declare-missing-class-properties.js

Reply:
probably needs tests as well. I could test it in the project i need it for, but unfortunately, I have been banging my head against a wall trying to figure out how to get it testable due to not being able to point at my fork in package.json because of lerna weirdnesses.

Reply:
do you have a way to test a branch in the wild without it being deployed to npm?